src/deploy-cromwell-on-azure/scripts/cromwell-application.conf

```diff
@@ -55,10 +55,10 @@ backend {
 }
 
 database {
-  db.url = "jdbc:mysql://{MySqlServerName}/{MySqlDatabaseName}?useSSL={MySqlUseSSL}&rewriteBatchedStatements=true&allowPublicKeyRetrieval=true"
-  db.user = "{MySqlUserLogin}"
-  db.password = "{MySqlUserPassword}"
-  db.driver = "com.mysql.cj.jdbc.Driver"
-  profile = "slick.jdbc.MySQLProfile$"
+  db.url = "jdbc:postgresql://{PostgreSqlServerName}/{PostgreSqlDatabaseName}?ssl={PostgreSqlUseSSL}&rewriteBatchedStatements=true&allowPublicKeyRetrieval=true"
```

Comment:
cromwellazure_triggerservice_1 isn't starting, I assume because this isn't quite configured correctly. Could use some insight as to whether we need rewriteBatchedStatements and allowPublicKeyRetrieval?

Reply:
cromwellazure_triggerservice_1 does not use the MySQL database so it's probably something else.  Docker logs is probably the best place to see the startup issue, can also try manually starting it and seeing what the error is.

rewriteBatchedStatements should be removed for postgreSQL, it's only a MySQL feature, and I don't think "allowPublicKeyRetrieval" is required since the connection should be SSL.  You could also cross-reference this connection string with what is given in the Azure Portal for this DB